docs/install/aws-install.md

```diff
@@ -72,24 +72,26 @@ You must fill in fields marked as mandatory (m) for the configuration to complet
 **Amazon EKS configuration**
 
 6. **Additional EKS admin ARN (IAM user)** (o) &mdash; add an additional admin user to the instance. Users can be added after the stack has been created. For more information see [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
-7. **Instance type** (m) &mdash; select from the dropdown list. The default is m5.2xlarge. For size considerations see [Amazon EC2 Instance Types](https://aws.amazon.com/ec2/instance-types/).
+7. **Instance type** (m) &mdash; select from the dropdown list. The default is m5.4xlarge. For size considerations see [Amazon EC2 Instance Types](https://aws.amazon.com/ec2/instance-types/).
 8. **Maximum Number of Nodes** (m) &mdash; maximum number of nodes in the cluster. The minimum value of this parameter must match the number of **Availability Zones**. The number of nodes combined with the **Instance type** determines the AWS infrastructure cost.
 
 **Amazon EC2 configuration**
 
-9. **SSH key name** (m) &mdash; select from the stored keys in the dropdown. The list is based on the SSH keys that are in your account. For more information about SSH Keys see [Amazon EC2 key pairs and Linux instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
+9. **SSH key name** (o) &mdash; Users who wish to access the EC2 instance via SSH can enter an existing key. If left empty, it is possible to access the EC2 instance using the AWS Systems Manager Session Manager. For more information about SSH Keys see [Amazon EC2 key pairs and Linux instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
 
 10. **Provision bastion host** (m) &mdash; create a bastion host for SSH access to the Kubernetes nodes. The default is enabled. This allows ssh access to your EKS EC2 instances through a public IP.
 
 **Iguazio MLRun configuration**
 
+<a id="route53_config" />
+
 11. **Route 53 hosted DNS domain** (m) &mdash; Enter the name of your registered Route53 domain. **Only route53 domains are acceptable.**
 
 12. **The URL of your REDIS database** (o) &mdash; This is only required if you're using Redis with the online feature store. See [how to configure the online feature store](#configure-online-feature-store) for more details.
 
 **Other parameters**
 
-13. **MLrunCeVersion** (m) &mdash; the MLRun Community Edition version to install. Leave the default value for the latest CE release.
+13. **MLRunCE Helm Chart version** (m) &mdash; the MLRun Community Edition version to install. Leave the default value for the latest CE release.
```

Comment:
```suggestion
13. **MLRun CE Helm Chart version** (m) &mdash; the MLRun Community Edition version to install. Leave the default value for the latest CE release.
```

I think it is better to separate the ce from mlrun when mentioning the product name